public/css/nhsuk-ie6.css

```diff
@@ -1,28 +1,28 @@
 @charset "UTF-8";
 @font-face {
   font-family: "FS Me";
-  src: url("/* @echo FONT_CDN_PATH */assets/fonts/fsme/FSMeWeb-Light.eot?") format("eot"), url("/* @echo FONT_CDN_PATH */assets/fonts/fsme/FSMeWeb-Light.woff") format("woff");
+  src: url("https://dhrlmnmyf2njb.cloudfront.net/assets/fonts/fsme/FSMeWeb-Light.eot?") format("eot"), url("https://dhrlmnmyf2njb.cloudfront.net/assets/fonts/fsme/FSMeWeb-Light.woff") format("woff");
```

Comment:
Reading the thread I expected them to be served from something like /fonts (i.e. a copy we hold similar to how we handle css).